metricbeat/docs/modules/aws.asciidoc

```diff
@@ -158,6 +136,9 @@ for a list of AWS services that publish metrics to CloudWatch.
 `period` for `rds` metricset is recommended to be `60s` or multiples of `60s` because Amazon RDS sends metrics and
 dimensions to Amazon CloudWatch every minute.
 
+[id="aws-credentials-config"]
+include::../../../../libbeat/docs/aws-credentials-config.asciidoc[]
```

Comment:
After you've added the attribute that I describe earlier, change this to:

`include::{libbeat-xpack-dir}/docs/aws-credentials-config.asciidoc[]`